auto_cpufreq/bin/auto_cpufreq.py

```diff
@@ -231,15 +161,14 @@ def config_info_dialog():
                 sys.argv.remove("--update")
                 if len(sys.argv) == 2: custom_dir = sys.argv[1] 
                     
-            if os.getenv("PKG_MARKER") == "SNAP":
+            if IS_INSTALLED_WITH_SNAP:
                 print("Detected auto-cpufreq was installed using snap")
                 # refresh snap directly using this command
                 # path wont work in this case
 
                 print("Please update using snap package manager, i.e: `sudo snap refresh auto-cpufreq`.")
                 #check for AUR 
-            elif subprocess.run(["bash", "-c", "command -v pacman >/dev/null 2>&1"]).returncode == 0 and subprocess.run(["bash", "-c", "pacman -Q auto-cpufreq >/dev/null 2>&1"]).returncode == 0:
-                print("Arch-based distribution with AUR support detected. Please refresh auto-cpufreq using your AUR helper.")
+            elif IS_INSTALLED_WITH_AUR: print("Arch-based distribution with AUR support detected. Please refresh auto-cpufreq using your AUR helper.")
```

Comment:
@shadeyg56 since IIRC you run Arch, can you confirm this line still works on your side?

Reply:
I'm on Artix (Arch based) and it works to detect if `auto-cpufreq` has been installed with AUR